.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,94 @@
+###########################################################################
+# Integration test suite definition for IRIDA Uploader using GitHub Actions
+###########################################################################
+
+name: Integration Tests
+
+on:
+  pull_request: # Run on all pull requests
+  push:
+    branches:   # Run on any push to development or master
+      - development
+      - master
+  schedule:     # Run weekly on development and master
+    - cron: 0 2 * * 1
+      branches: development
+    - cron: 0 2 * * 1
+      branches: master
+
+
+jobs:
+  build-unittests:
+    runs-on: ubuntu-20.04 #See pre-installed software at https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md
+    strategy:
+      fail-fast: False #Setting so that if one of the test suites fail, the other will continue
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2 #Checkout the project from git
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Run Unit Tests
+      run: make unittests
+
+  build-pep8:
+    runs-on: ubuntu-20.04 #See pre-installed software at https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md
+
+    steps:
+      - uses: actions/checkout@v2 #Checkout the project from git
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Run pep8
+        run: make pep8
+
+  build-irida-integration:
+    runs-on: ubuntu-20.04 #See pre-installed software at https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md
+
+    env:
+      MYSQL_PORT: 3306
+      MYSQL_USER: test
+      MYSQL_PASSWORD: test
+      MYSQL_DATABASE: irida_uploader_test
+      MYSQL_HOST: 127.0.0.1
+      MYSQL_ROOT_PASSWORD: password
+
+    strategy:
+      fail-fast: False #Setting so that if one of the test suites fail, the other will continue
+      matrix:
+        branch: ['master','development']
+
+    steps:
+      - uses: actions/checkout@v2 #Checkout the project from git
+      - name: Shutdown Ubuntu MySQL (SUDO)
+        run: sudo service mysql stop # Shutdown the Default MySQL, "sudo" is necessary, please not remove it
```

Comment:
Try removing this and running the tests to see if this is necessary.  From my reading it looks like mysql is stopped by default in the virtual environment.

Reply:
Yep, turns out it wasn't needed.